python/tests/test_metadata.py

```diff
@@ -542,6 +542,26 @@ def test_bad_type_error(self):
         ):
             ms.validate_and_encode_row(tskit.TableCollection(1))
 
+    def test_skip_validation(self):
+        ms = tskit.MetadataSchema({"codec": "json"})
```

Comment:
I guess mock.patch would be a bit shorter/more idiomatic here?
```python
with mock.patch(ms._validate_row) as mocked_validate:
     ms.validate_and_encode_row({})]
     assert mocked_validate.call_count == 0
```
something along these lines should do it.

Reply:
Done